erpnext/accounts/party.py

```diff
@@ -463,8 +463,8 @@ def get_timeline_data(doctype, name):
 	after = add_years(None, -1).strftime('%Y-%m-%d')
 	group_by='group by date(creation)'
 
-	data = get_communication_data(doctype, name, after=after, group_by='group by date(`tabCommunication`.creation)',
-		fields='date(`tabCommunication`.creation), count(`tabCommunication`.name)',as_dict=False)
+	data = get_communication_data(doctype, name, after=after, group_by='group by date(C.creation)',
```

Comment:
Change 
fields to `fields='date(C.creation) AS creation, count(C.name)'`
group_by to `group_by='GROUP BY date(creation)'`

Reply:
Fixed!